plugins/webhooks/index.js

```diff
@@ -836,8 +837,14 @@ exports.register = (server, options, next) => {
                         skipMessage = 'Skipping due to the commit message: [skip ci]';
                     }
 
+                    if (action === 'push' && !skipMessage && parsed.commitAuthors) {
```

Comment:
can probably get `commitAuthors` var on line 829 along with all the other keys in `parsed`